README.md

```diff
@@ -1,2 +1,127 @@
-# Africa-Tech-event
-Africa innovation tech tour
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Capstone 1</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Clone](#clone)
+  - [Run](#run)
+- [👥 Authors](#authors)
+- [🙏 Acknowledgements](#acknowledgements)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone 1 <a name="AFRICA TECH INNOVATION"></a>
+
+> This is the first capstone project in microverse course.
+
```

Comment:
- [ ]  **Congratulations on making a professional README** :medal_sports: 
I think it would be better to remove all the comments and guides in your readme file to make it more professional :joy: